service/src/main/java/bio/terra/drshub/config/DrsHubConfigInterface.java

```diff
@@ -30,4 +32,8 @@ public interface DrsHubConfigInterface {
   Integer getPencilsDownSeconds();
 
   Integer asyncThreads();
+
+  // If this is true, then we will track calls to the Bard API in MixPanel in addition to the
```

Comment:
Nit: this might be nice as a docstring instead, I think that might make the comment more visible when viewing references in IDEs.

Reply:
Also, is there a reason that we've named this slightly off from the existing convention of `pushToMixpanel`?

Reply:
Nope.  I'll update!

Reply:
Is this ever going to be `true`?

Reply:
Likely no, but it made it easier to test so I left it in as a potential knob